resources/design-document.md

```diff
@@ -0,0 +1,212 @@
+# Design Document
+
+## Jumanji Jamz
+
+## 1. Problem Statement
+
+Managing music charts during gigs downtown has been a challenge for our band. We need an app to organize, add, and edit charts seamlessly. This app will centralize all our charts, ensuring easy access from any device, making our performances smoother and more professional.
+
+## 2. Top Questions to Resolve
+
+1. How many tables we need
+2. Design for UI
+3. How frequently should we cache to  save time in between songs (create a Set Order?)
+
+## 3. Use Cases
```

Comment:
It's not critical, but I'd consider ordering these in a way such that they build on each other. E.g., before you can search  for charts or change the key, you need to make the setlist, create the chart, etc.  So that as you're reading through the user stories they each make sense in context of the others.